### ISSUE TITLE
How to add the audit code in template.md to a reST file?

### ISSUE THREAD

**Issue by orbeckst** [numfocus/templates#2]
In PR https://github.com/MDAnalysis/mdanalysis/pull/2467 we added NumFOCUS fiscal sponsorship information to the MDAnalysis repository README.rst, based on https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md

The template file contains a line `[//]: # (numfocus-fiscal-sponsor-attribution)` that we ought to include. However, it is not clear to us how to do this for a restructured text file, given that the template is in Markdown. 

For right now we included a [reST internal hyperlink target](https://docutils.sourceforge.io/docs/user/rst/quickref.html#internal-hyperlink-targets) in [README.rst](https://raw.githubusercontent.com/MDAnalysis/mdanalysis/develop/README.rst) that looks like this
```reST
.. _numfocus-fiscal-sponsor-attribution:
```

We would like to have some guidance on what you would like us to do to comply with the NumFOCUS audit policies.

**Comment by orbeckst** [comment]
cc @lisamartin72 (from https://github.com/MDAnalysis/mdanalysis/issues/2506)

**Comment by nicole-numfocus** [comment]
@sbrice Can you please explain what "sustainability audits" is referring to here? To my knowledge NumFOCUS is not using this tag to track anything currently.

**Comment by samdbrice** [comment]
Hi @orbeckst - thanks for updating your README with the fiscal sponsorship attribution. The hyperlink target works perfectly fine as we'd simply be grepping the text for that keyword.

@nicole-numfocus - Last summer we looked into ways to have projects tag their files that would enable us to automate checks (i.e. sustainability audits) for certain items deemed critical based on surveys from a recent summit (e.g. Donate button on the website, first-time contributor tag on GitHub, etc). 

For the README templates, we included some basic tags that wouldn't show up from a reader's perspective but can be easily searched for by a program to generate a dashboard/report on which projects have all their info up to date.
